src/components/footer.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { MDBCol, MDBContainer, MDBRow, MDBFooter } from 'mdbreact';
+
+class Footer extends Component {
+    state = {
```

Comment:
This doesn't refer to `React States`, Then why name it `state`? You could use const variables for different elements here like `footerFirstCol`, `footerCols` etc.. separately.